packages/wouter-preact/package.json

```diff
@@ -35,12 +35,11 @@
       "default": "./esm/static-location.js"
     }
   },
-  "module": "esm/index.js",
-  "types": "types/ts3.9.4/index.d.ts",
+  "types": "types/index.d.ts",
   "typesVersions": {
```

Comment:
Could you check what versions of TS support type declaration paths defined in `exports`? I've got a feeling that we don't need `typesVersions` at all now.

Reply:
nope, `type: module` doesn't affect how typescript resolves types. `typesVersions` is used when `moduleResolution` is set to `node`, and the `type` field in `exports` is used when `moduleResolution` is `nodenext` or `node16` or `bundler`

Reply:
ok!